tiatoolbox/annotation/storage.py

```diff
@@ -1071,80 +1072,147 @@ def to_geodict(self) -> Dict[str, Any]:
         """Helper function to handle cases for dumping.
 
         Args:
-            fp:
+            full_path:
                 The file path or handle to dump to.
             file_fn(Callable):
-                The function to call when fp is a file handle.
+                The function to call when full_path is a file handle.
             none_fn(Callable):
-                The function to call when fp is None.
+                The function to call when full_path is None.
 
         Returns:
             Any:
                 The result of dump. Depends on the provided functions.
 
         """
-        if fp is not None:
+        if full_path is not None:
             # It is a file-like object, write to it
-            if hasattr(fp, "write"):
-                return file_fn(fp)
+            if hasattr(full_path, "write"):
+                return file_fn(full_path)
             # Turn a path into a file handle, then write to it
-            with open(fp, "w", encoding="utf-8") as file_handle:
+            with open(full_path, "w", encoding="utf-8") as file_handle:
                 return file_fn(file_handle)
         # Return as str or bytes if no handle/path is given
         return none_fn()
 
     @staticmethod
     def _load_cases(
-        fp: Union[IO, str, Path],
+        full_path: Union[IO, str, Path],
         string_fn: Callable[[Union[str, bytes]], Any],
         file_fn: Callable[[IO], Any],
     ) -> Any:
         with contextlib.suppress(OSError):
-            if isinstance(fp, (Path, str)) and Path(fp).exists():
-                with open(fp) as file_handle:
+            if isinstance(full_path, (Path, str)) and Path(full_path).exists():
+                with open(full_path) as file_handle:
                     return file_fn(file_handle)
-        if isinstance(fp, (str, bytes)):
-            return string_fn(fp)
-        if hasattr(fp, "read"):
-            return file_fn(fp)
+        if isinstance(full_path, (str, bytes)):
+            return string_fn(full_path)
+        if hasattr(full_path, "read"):
+            return file_fn(full_path)
         raise IOError("Invalid file handle or path.")
 
     @classmethod
-    def from_geojson(cls, fp: Union[IO, str]) -> "AnnotationStore":
-        geojson = cls._load_cases(
-            fp=fp,
+    def from_geojson(
+        cls,
+        full_path: Union[IO, str],
+        scale_factor: float = 1,
+        relative_to: Tuple[float, float] = None,
+    ) -> "AnnotationStore":
+        """Create a new database with annotations loaded from a geoJSON file.
+        Args:
+            full_path (Union[IO, str, Path]):
+                The file path or handle to load from.
+            scale_factor (float):
+                The scale factor to use when loading the annotations. All coordinates
+                will be multiplied by this factor to allow import of annotations saved
+                at non-baseline resolution.
+            relative_to [float, float]:
```

Comment:
```suggestion
            origin [float, float] = (0, 0):
```

I think 'origin' is slightly less ambiguous / more clear than 'relative_to' IMO.

Reply:
Origin would be clearer, i'll change it to that